src/stateless/selectbox.vue

```diff
@@ -58,7 +58,7 @@
 
                         <div v-for="option in group.options" :data-option-id="option.id" :class="getOptionCssStates(option) | prefix('selectbox__option--')"
                              :key="option.id" class="selectbox__option" @click="selectOption(option.id)">
-                            <div :class="getOptionCssStates(option) | prefix('selectbox__label--')" :title="option.label" class="selectbox__label">
+                            <div :class="getOptionCssStates(option) | prefix('selectbox__label--')" :style="showOptionMetadata ? { width: '70%' } : { width: '100%' }" :title="option.label" class="selectbox__label">
```

Comment:
Why not `option.metadata ? { width: '70%' } : { width: '100%' }` instead? I know it's not the same since different labels could then have different widths, but is that an actual problem?

Reply:
It doesn't look nice.